modules/mono/editor/GodotTools/GodotTools/Export/ExportPlugin.cs

```diff
@@ -361,7 +361,7 @@ private void _ExportBeginImpl(string[] features, bool isDebug, string path, long
                 }
 
                 var xcFrameworkPath = Path.Combine(GodotSharpDirs.ProjectBaseOutputPath, publishConfig.BuildConfig,
-                    $"{GodotSharpDirs.ProjectAssemblyName}.xcframework");
+                    $"{GodotSharpDirs.ProjectAssemblyName}_aot.xcframework");
```

Comment:
For the reference, this is the only changed line in the last push. Added to fix name conflict (https://github.com/godotengine/godot/pull/82729#pullrequestreview-1656972006) and should not have any effect on the rest of code.